### PR TITLE
Bottom Navigation View UI Test

### LINKED ISSUE #45: UI Tests for Navigation

### PR DESCRIPTION
**Pull request by cyrusdiego**
# Summary
- Added simple UI test for the Bottom Navigation View 
- Added an abstract class that will bypass login for future tests 

# TODO
- Testing the actual navigation graph  (I spent a few hours trying to get it setup but to no avail will try to continue with later into the project but I don't think it's a "mandatory" component to test as I think they just expect Unit and UI tests

Closes #45

### CODE REVIEW

File: app/src/androidTest/java/com/example/unlibrary/MockLogin.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public abstract class MockLogin {
+    private final String mEmail = "cdiego@ualberta.ca";
+    private final String mPassword = "password123";
```
Comment (calebschoepp) [inline app/src/androidTest/java/com/example/unlibrary/MockLogin.java:16]: This is just an actually valid email/password combo in our Firebase Auth right?

File: app/src/androidTest/java/com/example/unlibrary/MockLogin.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import com.google.firebase.auth.FirebaseAuth;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public abstract class MockLogin {
```
Comment (calebschoepp) [inline app/src/androidTest/java/com/example/unlibrary/MockLogin.java:14]: Nice! I like this pattern to avoid the login.

**Review by calebschoepp — APPROVED** [review]
Not very familiar with Espresso tests but these LGTM! :ship:

File: app/src/androidTest/java/com/example/unlibrary/MockLogin.java (hunk 1 above)
Comment (cyrusdiego) [reply app/src/androidTest/java/com/example/unlibrary/MockLogin.java:16]: yeah it's my account (so long as I don't change my user info), it's just so we can bypass the login

**Pull request merged**